index.html

```diff
@@ -63,23 +63,221 @@ <h1 class="primary">
+                            Gain+Glory
+                        </h3>
+                        <ul class="tags">
+                            <li class="tag">HTML</li>
+                            <li class="tag">JS</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">Ruby & Rails</li>
+                        </ul>
+                        <button class="butn" type="button" title="see-project">
+                            See Project
+                          </button>
+                    </div>
+                </li>
+            </ul>
+        </section>
+        <section class="about-me">
+            <div class="text-on-about">
+                <h2 class="primary-text">About me</h2>
+                <p class="supporting-text-about">
+                    I can help you build a product , <br> feature or website Look through <br> some of my work and experience! <br> If you like what you see and have <br> a project you need coded, <br>
+                    don’t hesitate to contact me.
+                </p>
+                <button class="butn" type="button" title="resume">
+                    Get my resume
+                  </button>
+            </div>
+            <ul class="card">
+                <li class="card-works-inactive">
+                    <img
+                    src="assets/images/language-icons.svg" alt="languages" />
+                  <h3 class="title">Languages</h3>
+                    <ul class="frame19-tags">
+                        <li class="tag">HTML</li>
+                        <li class="tag">JS</li>
+                        <li class="tag">CSS</li>
+                        <li class="tag">Ruby</li>
+                    </ul>
+                </li>
+                <li class="card-works-inactive">
+                    <img
+                    src="assets/images/farmework-icons.svg" alt="framworks" />
+                  <h3 class="title">Framworks</h3>
+                    <ul class="frame19-tags">
+                        <li class="tag">React.js</li>
+                        <li class="tag">Ruby and Rails</li>
+                        <li class="tag">RSpec</li>
+                        <li class="tag">Capybara</li>
+                        <li class="tag">Selenium</li>
+                    </ul>
+                </li>
+                <li class="card-works-inactive">
+                    <img
+                    src="assets/images/skill-icons.svg" alt="skills" />
+                  <h3 class="title">Skills</h3>
+                    <ul class="frame19-tags">
+                        <li class="tag">Database Management</li>
+                        <li class="tag">Version Control</li>
+                        <li class="tag">CLI</li>
+                        <li class="tag">Web Development</li>
+                        <li class="tag">API Design</li>
+                    </ul>
+                </li>
+            </ul>
```

Comment:
- [ Required ] I think it would be better if you made sure that the borders of the technology list are the same `size` and `color` as those in Figma so that your design matches the Figma design. The borders in your design differ significantly in `color` and `margin` ( especially the right-hand side margin ) from those in Figma please see the screenshot below for more details on this.

| Figma design | Your design |
| -------------------|-----------------|
|![Screenshot from 2023-06-03 03-29-41](https://github.com/yesuf023/Portfolio/assets/16392046/c822707e-881f-44d9-8997-db4619f38243)|![Screenshot from 2023-06-03 03-28-01](https://github.com/yesuf023/Portfolio/assets/16392046/6232ccbe-fa8e-41c8-962a-0980b21e7172)|